src/modules/user/dto/user-update.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { IsOptional, IsString } from 'class-validator'
+import { UserUpdate } from '../domain/user-update.domain'
+import { ApiPropertyOptional } from '@nestjs/swagger'
+
+export class UserUpdateDto {
+  @ApiPropertyOptional({
+    example: 'John Doe',
+    description: 'Full name of the user',
+  })
+  @IsOptional()
+  @IsString()
+  name?: string;
+
+  static toDomain(userUpdateDto: UserUpdateDto): UserUpdate {
+    return { ...userUpdateDto }
```

Comment:
Thiếu ;

Không được dùng ...

```suggestion
    return { 
        name: userUpdateDto.name
    };
```